ramls/schemas/profiles/transformations.json

```diff
@@ -27,8 +27,8 @@
       "type": "string",
       "$ref": "recordTypes.json"
     },
-    "hasSameTagInHoldings": {
-      "description": "The sign that means the current transformation intended for both item and holding",
+    "hasSameTagInItems": {
```

Comment:
This is a breaking change, it needs a change in UI as well as a migration script